### PR TITLE
Add (broken) cii experiment

### PR DESCRIPTION
**Pull request by BenJam**
Todo

* fix pg query syntax
* double check render
* add test

### CODE REVIEW

**Comment by BenJam** [comment]
@andrew would be good to understand what's happening with the db qry here. Tripping up on `projets_controller#digital_infrastructure` ln 144. But I'm certain this is due to `project.rb` ln 80 but I took the query for the rake task.

File: app/models/project.rb [hunk 1]
```diff
@@ -77,6 +77,11 @@ class Project < ApplicationRecord
                              .where('repositories.stargazers_count < 100')
                              .where('projects.dependent_repos_count > 1000') }
 
+  scope :digital_infrastructure, -> { not_removed
+                             .with_dependent_repos
+                             .with_repo
+                             .where('projects.dependent_repos_count DESC NULLS LAST').limit(1000)}
```
Comment (andrew) [inline app/models/project.rb]: This should be .order, rather than .where at the start of this line

File: app/controllers/projects_controller.rb [hunk 2]
```diff
@@ -138,11 +138,18 @@ def sync
...
+    @platforms = orginal_scope.group('projects.platform').count.reject{|k,_v| k.blank? }.sort_by{|_k,v| v }.reverse.first(20)
+  end
+
   def unseen_infrastructure
     orginal_scope = Project.unsung_heroes
     scope = current_platform.present? ? orginal_scope.platform(current_platform) : orginal_scope
     @projects = scope.order('projects.dependent_repos_count DESC').paginate(page: page_number)
-    @platforms = orginal_scope.group('projects.platform').count.reject{|k,_v| k.blank? }.sort_by{|_k,v| v }.reverse.first(20)
+    @platforms = orginal_scope.group('projects.platform').count.reject{|k,_v|÷ k.blank? }.sort_by{|_k,v| v }.reverse.first(20)
   end
```
Comment (andrew) [inline app/controllers/projects_controller.rb]: Weird percentage symbol on this line that needs to be removed

File: app/models/project.rb (hunk 1 above)
Comment (BenJam) [inline app/models/project.rb]: 👍

File: app/controllers/projects_controller.rb (hunk 2 above)
Comment (BenJam) [inline app/controllers/projects_controller.rb]: 🥇

**Comment by BenJam** [comment]
Tak!

**Pull request merged**